packages/frontmatter/src/frontmatter/types.ts

```diff
@@ -93,11 +93,20 @@ export type Export = {
   output?: string;
 } & Record<string, any>;
 
+export type SiteDesign = {
+  hide_authors?: boolean;
+  hide_toc?: boolean;
+  hide_title_block?: boolean;
+  hide_outline?: boolean;
+  hide_footer_links?: boolean;
+};
+
 export type SiteFrontmatter = {
   title?: string;
   description?: string;
   venue?: Venue;
   keywords?: string[];
+  design?: SiteDesign;
```

Comment:
Probably should remove `design` from `SiteManifest` if we are adding it here, just to not have the duplication.

Reply:
Oh - I think the only reason this works is because `design` was not removed from `SiteManifest`... otherwise `SITE_FRONTMATTER_KEYS` without design would filter it out...